packages/core/src/stylable-processor.ts

```diff
@@ -287,6 +313,11 @@ export class StylableProcessor implements FeatureContext {
                         rule,
                         walkContext: nodeContext,
                     });
+                } else if (node.value.startsWith('--')) {
```

Comment:
we have a function for this check

Reply:
we don't. there is a similar function that validates the name of a custom property. nothing like that for custom selector